zulip_bots/zulip_bots/bots/lunch/lunch.py

```diff
@@ -0,0 +1,220 @@
+            if item_str[-1] == ',':
+                item_str += item
+            else:
+                item_str += ' ' + item
+
+    return item_str[1:].split(',')
+
+# Copies user list
+def copy_user_list(original_user, user):
+    # Verification
+    user_not_found = find_user(original_user, True)
+    if user_not_found[0]:
+        return user_not_found[1]
+    find_user(user, False)
+
+    # Copies list
+    ORDERS[user].extend(ORDERS[original_user])
+
+    return "**Here's the new** @**" + user + '** **list of items:** :wink:\n' + ', '.join(ORDERS[user])
+
+# Adds an item to a list
+def add_to_user_list(user, items):
+    # Verification
+    find_user(user, False)
+
+    # Add items
+    items = item_interpreter(items)
+    ORDERS[user].extend(items)
+
+    return "**Here's the new** @**" + user + '** **list of items:** :wink:\n' + ', '.join(ORDERS[user])
+
+# Removes an item from a list
+def remove_from_user_list(user, items):
+    # Verification
+    user_not_found = find_user(user, True)
+    if user_not_found[0]:
+        return user_not_found[1]
+
+    # Removes items
+    items = ' '.join(items)
+    # Verify if it's an array fo items or not
+    if ',' in items:
+        items = items.split(', ')
+        for item in items:
+            try:
+                ORDERS[user].remove(item)
+            except ValueError:
+                return "@**" + user + "** ** doesn't have " + item + " in the list** :neutral_face:"
+    else:
+        try:
+            ORDERS[user].remove(items)
+        except ValueError:
+            return "@**" + user + "** ** doesn't have " + items + " in the list** :neutral_face:"
+
+    return "**Here's the new** @**" + user + '** **list of items:** :wink:\n' + ', '.join(ORDERS[user])
+
+# Removes the list of items
+def remove_all(user):
+    # Verification
+    user_not_found = find_user(user, True)
+    if user_not_found[0]:
+        return user_not_found[1]
+
+    # Removes user
+    del ORDERS[user]
+
+    return 'Cleared @**' + user + '** list! :ok_hand:'
+
+# Shows all orders
+def show_all_orders():
+    order_str = ''
+    for order in ORDERS:
+        order_str = order + ' **wants** ' + ', '.join(ORDERS[order])
+
+    return "**Here's the list of orders:**\n " + order_str
+
+# Shows a specific order
+def show_order(user):
+    # Verification
+    user_not_found = find_user(user, True)
+    if user_not_found[0]:
+        return user_not_found[1]
+
+    return "**Here's** @**" + user + '** **list:**\n' + ', '.join(ORDERS[user])
+
+def random_user(verb):
+    users = list(ORDERS.keys())
+
+    verb = ''.join(item_interpreter(verb))
+    if verb[-1] == '?':
+        verb = verb[:-1]
+
+    # Checks if the ORDERS.keys() isn't empty
+    if len(users) == 0:
+        return "**No one wants to " + verb + "**. :anguished:"
+
+    # Selects the user
+    user = random.choice(users)
+
+    return "**I think @**" + user + "** should " + verb + ".** :stuck_out_tongue:"
+
+handler_class = LunchBotHandler
```

Comment:
I think this goes for all the functions for this module that most of the comments are redundant and describe what the code itself sufficiently indicates. I would recommend removing all of the comments! :)

Reply:
Removed, nice to know the code explains itself :)